packages/paste-core/components/form-pill-group/package.json

```diff
@@ -58,6 +59,7 @@
     "@twilio-paste/style-props": "^9.1.0",
     "@twilio-paste/styling-library": "^3.0.0",
     "@twilio-paste/theme": "^11.0.0",
```

Comment:
any reason why `peerDependencies` has 14.0.0 version but `devDependencies` has 14.1.0 version of `@twilio-paste/truncate`?

Reply:
This was the same versions for each in other components that used it to be consistent. Should resolve locally anyway

Reply:
cool cool